scripts/deploy/deploy-dashboard.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
```

Comment:
Add a top-level comment mentioning what this script does ("Deploys dashboard app to Firebase, both hosting and functions").